selectors/Overview.bs

```diff
@@ -2031,6 +2036,49 @@ The Input Focus Pseudo-class: '':focus''</h3>
 	the same ways must apply to elements matching '':focus'' as well,
 	except that the parent of an element that matches '':focus'' must not match '':focus''.
 
+<h3 id="the-focusring-pseudo">
+The Input Focusring Pseudo-class: '':focusring''</h3>
+
+	The <dfn id="focusring-pseudo'>:focusring</dfn> pseudo-class is similar to the
+	:focus pseudo-class, but it only matches an element if the element is
+	currently focused and a focus ring or other indicator should be drawn for that
+	element. If :focusring matches, then :focus also matches, but the converse is
+	not always true - it depends on whether the user agent has focus ring drawing
+	enabled and how the element was focused. Whether the user agent has focus ring
+	drawing enabled can depend on things like the settings of the operating system
+	the user is using, so the precise behavior of this pseudo-class can vary from
+	platform to platform depending on each platform's particular focus best
+	practices (defaults) or user modified settings.
+
+	<div class='example'>
```

Comment:
I think the spec uses single quotes for these attributes https://github.com/robdodson/csswg-drafts/blob/06b54b7e135b6b823d8d9532c8b1bd21b9164ab5/selectors/Overview.bs#L695